src/Framework/Validation/Rules/Currency.php

```diff
@@ -0,0 +1,235 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Give\Framework\Validation\Rules;
+
+use Closure;
+use Give\Framework\Validation\Contracts\ValidatesOnFrontEnd;
+use Give\Framework\Validation\Contracts\ValidationRule;
+
+class Currency implements ValidationRule, ValidatesOnFrontEnd
+{
+    /**
+     * @inheritDoc
+     *
+     * @unreleased
+     */
+    public static function id(): string
+    {
+        return 'currency';
+    }
+
+    /**
+     * @inheritDoc
+     *
+     * @unreleased
+     */
+    public static function fromString(string $options = null): ValidationRule
+    {
+        return new self();
+    }
+
+    /**
+     * @unreleased
+     *
+     * @return string[]
+     */
+    public static function currencyCodes(): array
+    {
+        static $codes = null;
+
+        if ($codes === null) {
+            $codes = [
```

Comment:
Out of curiosity where did you get this list from?  The first thing I noticed is it's not alphabetical 😅

Reply:
Internet. 😂 😂

Reply:
Better? 😁 bac55aea9